build/Tests/UnitTests/Starboard/Starboard.UnitTests.vcxproj

```diff
@@ -1,247 +1,250 @@
+    </ClCompile>
+    <Link>
+      <SubSystem>Console</SubSystem>
+      <GenerateDebugInformation>true</GenerateDebugInformation>
+      <EnableCOMDATFolding>true</EnableCOMDATFolding>
+      <OptimizeReferences>true</OptimizeReferences>
+      <AdditionalDependencies>RTObjCInterop.lib;ObjCUWP.lib;mincore.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AppContainer>false</AppContainer>
+    </Link>
+    <ClangCompile>
+      <IncludePaths>$(StarboardBasePath)\Frameworks\include;$(StarboardBasePath)\tests\frameworks\include;$(StarboardBasePath)\tests\frameworks\gtest;$(StarboardBasePath)\tests\frameworks\gtest\include;$(StarboardBasePath)\;$(StarboardBasePath)\tests\unittests\Tests.Shared;%(AdditionalIncludeDirectories)</IncludePaths>
+      <CompileAs>CompileAsObjCpp</CompileAs>
+      <PreprocessorDefinitions>SB_IMPEXP=;NO_STUBS;_CRT_SECURE_NO_WARNINGS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>
+      <OptimizationLevel>Full</OptimizationLevel>
+    </ClangCompile>
+  </ItemDefinitionGroup>
+  <ItemGroup>
+    <ClCompile Include="$(StarboardBasePath)\tests\unittests\Framework\Framework.cpp" />
+    <ClCompile Include="$(StarboardBasePath)\tests\unittests\EntryPoint.cpp" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClangCompile Include="..\..\..\..\tests\unittests\Starboard\AutoIdTests_ARC.mm" />
+    <ClangCompile Include="..\..\..\..\tests\unittests\Starboard\AutoIdTests_NoARC.mm" />
+    <ClangCompile Include="..\..\..\..\tests\unittests\Starboard\CommonCryptoTests.mm" />
+    <ClangCompile Include="..\..\..\..\tests\unittests\Starboard\LifetimeCounting.mm" />
+    <ClangCompile Include="..\..\..\..\tests\unittests\Starboard\ErrorHandling.mm" />
+    <ClangCompile Include="..\..\..\..\tests\unittests\Starboard\objcrt_assoc.mm" />
+    <ClangCompile Include="..\..\..\..\tests\unittests\Starboard\objcrt_runtime.mm" />
+    <ClangCompile Include="..\..\..\..\tests\unittests\Starboard\objcrt_sanity.m" />
+    <ClangCompile Include="..\..\..\..\tests\unittests\Starboard\PthreadTests.mm" />
+    <ClangCompile Include="..\..\..\..\tests\unittests\Starboard\ProjectionsTests.mm" />
+    <ClangCompile Include="..\..\..\..\tests\unittests\Starboard\CommonCryptoTests.m">
+      <CompileAs>CompileAsObjc</CompileAs>
+    </ClangCompile>
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="..\..\..\..\tests\unittests\Starboard\LifetimeCounting.h" />
+  </ItemGroup>
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
+  <ImportGroup Label="ExtensionTargets">
+    <Import Project="$(StarboardBasePath)\msvc\starboard-cmdline.targets" />
+  </ImportGroup>
+</Project>
```

Comment:
github is showing this file as 100% changed; has its encoding or line ending style changed?